airbyte-integrations/connectors/source-freshdesk/source_freshdesk/streams.py

```diff
@@ -0,0 +1,254 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+
+from abc import ABC
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional
+from urllib import parse
+
+import pendulum
+import requests
+import re
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.sources.streams.core import IncrementalMixin
+from airbyte_cdk.sources.streams.http import HttpStream
+from requests.auth import AuthBase
+from source_freshdesk.utils import CallCredit
+
+
+class FreshdeskStream(HttpStream, ABC):
+    """Basic stream API that allows to iterate over entities"""
+    call_credit = 1  # see https://developers.freshdesk.com/api/#embedding
```

Comment:
This value never changes, I'd prefer to decrement from `1` rather than decrementing from this variable if it never changes.

Reply:
I double checked and it was actually being used with a different value in the `Tickets` stream. I've updated it accordingly.